contrib/guix/patches/glibc-2.27-riscv64-Use-__has_include-to-include-asm-syscalls.h.patch

```diff
@@ -4,6 +4,8 @@ See also: http://lists.busybox.net/pipermail/buildroot/2020-July/590376.html.
 
 https://sourceware.org/git/?p=glibc.git;a=commit;h=0b9c84906f653978fb8768c7ebd0ee14a47e662e
 
+This patch can be dropped when we are building with glibc 2.28+.
```

Comment:
https://sourceware.org/bugzilla/show_bug.cgi?id=24022:
> Target Milestone: | 2.29

Reply:
> https://sourceware.org/bugzilla/show_bug.cgi?id=24022:

Again. The thread says it was backported to 2.28, which is what matters for us.